src/components/accordion.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { Collapse, Button, CardBody, Card } from 'reactstrap';
+import styled, { css } from "react-emotion";
+
+// CSS
+
+const StyledButton = styled(Button)`
+  font-size: 14px;
+`
+
+const StyledCard = styled(Card)`
+  font-size: 13px;
+  border-width: 3px;
+`
+
+const StyledCardBody = styled(CardBody)`
+  padding: 10px;
+`
+
+class Accordion extends Component {
+  constructor(props) {
+    super(props);
+    this.toggle = this.toggle.bind(this);
+    this.state = { collapse: false };
+  }
+
+  toggle() {
+    this.setState(state => ({ collapse: !state.collapse }));
+  }
+
+  render() {
+    return (
+      <div>
+        <StyledButton color="link" onClick={this.toggle}>{this.props.title}</StyledButton>
+        <Collapse isOpen={this.state.collapse}>
+          <StyledCard>
+            <StyledCardBody>
+              {this.props.body}
+            </StyledCardBody>
+          </StyledCard>
+        </Collapse>
+      </div>
+    );
+  }
+}
+
+export default Accordion;
```

Comment:
What controls the speed of this accordion?

Reply:
it should be all built into the `Collapse` component I think. Can't really find the source code. Do you think we need to change the speed of the accordion or are you just curious? 

https://react-bootstrap.github.io/utilities/transitions/
https://getbootstrap.com/docs/4.3/components/collapse/#accordion-example

Reply:
So `Collapse` is wrapped in a `Transition` component and you can pass in the wrapper's props and probably control the speed with the timout prop:
http://reactcommunity.org/react-transition-group/transition